benchmarks/benchmarks.h

```diff
@@ -501,6 +501,7 @@ static void BM_Swap(benchmark::State& state)
 
 ////////////////////////////////////////////////////////////////////////////////////////
 // This is where the benchmarks are programmatically registered .
+template <typename Str>
 void RegisterBenchmarks(const char* classname) {
```

Comment:
That is soooooooooooooooo weird. I do NOT know which compiler you are using.

I strongly think there is NO compiler that can compile this code.

Reply:
I can confirm that I've compiled this with Clang 13.1.6 and MSVC 14.32.31326

Reply:
Thanks for catching this though! I had this template argument in my development version and I forgot to incorporate it here.